server/cassandra.js

```diff
@@ -0,0 +1,79 @@
+const cassandra = require('cassandra-driver');
+
+const client = new cassandra.Client({
+  contactPoints: ['localhost:9042'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'reviews'
+});
+
+const getAllReviews = () => {
```

Comment:
You can make use of `util`'s `promisify` function to eliminate the need for callbacks in each of these queries.

Reply:
Forgot that Cassandra driver returns promises- fixed!